### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by SinshiSmith**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["next/babel"]
+}
```
Comment (Mathspy) [inline .babelrc]: newline

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true,
+  },
+  extends: ["eslint:recommended", "plugin:react/recommended"],
```
Comment (Mathspy) [inline .eslintrc.js]: [eslint-plugin-react-hooks](http://npm.im/eslint-plugin-react-hooks)

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
+.next
```
Comment (Mathspy) [inline .gitignore]: newline

File: .prettierignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+build
+coverage
+.next
```
Comment (Mathspy) [inline .prettierignore]: newline

**Review by Mathspy — APPROVED** [review]
Looks good!

File: .babelrc (hunk 1 above)
Comment (SinshiSmith) [reply .babelrc]: prettier is gonna kill me

**Pull request merged**